src/config/links.ts

```diff
@@ -6,11 +6,17 @@ const INTERLAY_GITHUB_LINK = 'https://github.com/interlay';
 const INTERLAY_TWITTER_LINK = 'https://twitter.com/interlayHQ';
 const INTERLAY_DOCS_LINK = 'https://docs.interlay.io';
 
+// TODO: placeholders
+const INTERLAY_TERMS_AND_CONDITIONS_LINK = '';
+const KINTSUGI_TERMS_AND_CONDITIONS_LINK = '';
```

Comment:
@alexeiZamyatin 
Once we have put the links here, the other parts should be working as expected.